2-python-fundamentals-i.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
_Commented on notebook `2-python-fundamentals-i.ipynb` Cell 153 Line 5_
```markdown
### The infinite loop

Beware the infamous infinite loop! Consider the following example, where the condition will never become false:

**NOTE** to stop this, you will have to press <kbd>CTRL</kbd> + <kbd>c</kbd> in the interpreter, or the "STOP" button in jupyter lab / notebook
```
We're using VSCode for the course right? Maybe replace the jupyter instruction with the vscode equivalent?